doc/VIMUSERS.org

```diff
@@ -330,36 +329,46 @@ builtin command to generate the layer boilerplate:
 
 The =packages.el= file contains a list of packages that you can install in the
 variable =<layer-name>-packages=. Any package that is available on the [[http:melpa.org][MELPA]]
-repository can be added to the list. You can also exclude packages by adding
-them to =<layer-name>-excluded-packages=. Each package requires a function to
-initialize it. The function /must/ be named with this pattern:
-=<layer-name>/init-<package-name>=. This function contains configuration for the
-package. It would look like this:
+repository can be added to the list. A list can also exclude packages using the
+=:excluded= property.
+Each package requires a function to initialize it. The function /must/ be named
+with this pattern: =<layer-name>/configure-<package-name>=. This function contains
+configuration for the package. It would look like this:
 
 #+begin_src emacs-lisp
-  (setq layer-name-packages '(example-package))
-  (defun layer-name/init-example-package ()
+  (setq layer-name-packages '(example-package
+                              ;; This layer uninstalls example-package-2
+                              ;; by setting the :excluded property to true (t)
+                              (example-package-2 :excluded t)))
+  (defun layer-name/configure-example-package ()
     ;; Configuration for example-package goes here
     )
 #+end_src
 
-If something is not available on MELPA, you must use an extension. Extension
-configuration is done in the =extensions.el= file. Each extension must be placed
-in its own folder inside the =extensions= folder. Extensions can be declared
-using the =<layer-name>-<pre/post>-extensions= variables. =pre= extensions are
-loaded before the packages and =post= extensions are loaded after. The name of
-the extension is the name of the folder it is in. Using the above example
-structure, the extensions would be activated like so:
+**Note**: Only one layer can have a configure function for a package. If you
+want to override the configuration of a package in another layer, see **link**.
+**TODO**: Add link to relevant section in LAYERS.org
+
+If a package is not available on MELPA, you must use an local package. Each
+local package must be placed in its own folder inside the =local= folder. The
+name of the local package is the name of the folder it is in. Local packages can
+be declared in the =<layer-name-packages= variable using the =:local= property
```

Comment:
this is `:location` instead of `:local` because a package can have only one location, that is `elpa`, `local` or `recipe` (soon to be implemented)